apps-rendering/src/components/Callout/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { css, ThemeProvider } from '@emotion/react';
+import type { FormField } from '@guardian/apps-rendering-api-models/formField';
+import type { ArticleFormat } from '@guardian/libs';
+import { remSpace } from '@guardian/source-foundations';
+import { ExpandingWrapper } from '@guardian/source-react-components-development-kitchen';
+import { isElement } from 'lib';
+import type { FC, ReactElement } from 'react';
+import { createElement as h } from 'react';
+import CalloutBlock from './calloutBlock';
+import { DeadlineDate, Highlight, isCalloutActive } from './deadlineDate';
+import { getTheme } from './theme';
+
+export interface CalloutProps {
+	heading: string;
+	formId: number;
+	formFields: FormField[];
+	format: ArticleFormat;
+	description?: DocumentFragment;
+	isNonCollapsible: boolean;
+	activeUntil?: number;
+	name: string;
+}
+
+const Callout: FC<CalloutProps> = ({
+	heading,
+	description,
+	formId,
+	formFields,
+	format,
+	isNonCollapsible,
+	activeUntil,
+	name,
+}): ReactElement => {
+	const isActive = isCalloutActive(activeUntil);
+
+	if (!isActive && isNonCollapsible) {
+		return (
+			<Highlight>
+				This form has been deactivated and is closed to any further
+				submissions.
+			</Highlight>
+		);
+	} else if (!isActive && !isNonCollapsible) {
+		return <></>;
```

Comment:
If the callout is not active, how come we don't return the highlight again here?

Also would it be possible to return null here or do we need the empty tags?

Reply:
It the callout is collapsable then its an embedded callout, and after talking with Ben and Bella we decided it would be better to not show anything rather than an embedded highlight about a callout thats closed.

I'll see if null works though!

Reply:
nice, thanks for the explainer :)